### PR TITLE
fix block interval calculation

### PR DESCRIPTION
**Pull request by 0xAshish**
fix for https://github.com/maticnetwork/contracts/issues/212

### CODE REVIEW

File: contracts/root/RootChain.sol [hunk 1]
```diff
@@ -53,7 +53,7 @@ contract RootChain is RootChainStorage, IRootChain {
         );
         // blockInterval, voteHash, stateRoot, sigs
         uint256 _reward = stakeManager.checkSignatures(
-            headerBlock.end.sub(headerBlock.start),
+            headerBlock.end.sub(headerBlock.start.add(1)),
```
Comment (jdkanani) [inline contracts/root/RootChain.sol]:
```suggestion
            headerBlock.end.sub(headerBlock.start).add(1),
```

Comment (0xAshish) [reply contracts/root/RootChain.sol]: Thanks fixed.

File: contracts/staking/StakingInfo.sol [hunk 2]
```diff
@@ -99,6 +99,11 @@ contract StakingInfo {
         uint256 indexed rewards,
         uint256 tokens
     );
+    event DelReStaked(
```
Comment (atvanguard) [inline contracts/staking/StakingInfo.sol]: Not clear from the name what this event does.

Comment (0xAshish) [reply contracts/staking/StakingInfo.sol]: Del is a prefix for delegation related events and `ReStaked` is for `reStaking`.

Comment (atvanguard) [reply contracts/staking/StakingInfo.sol]: Clearer name would have helped. Approving...

**Pull request merged**